core/app/models/spree/tax_calculator/default.rb

```diff
@@ -0,0 +1,46 @@
+module Spree
+  module TaxCalculator
+    class Default
+      include Spree::Tax::TaxHelpers
+
+      def initialize(order)
+        @order = order
+      end
+
+      def calculate
+        Spree::Tax::Output::Order.new(
+          id: order.id,
+          line_items: line_item_rates,
+          shipments: shipment_rates
```

Comment:
Similarly here, with `shipment_taxes`. 

By the way: We had a discussion earlier about the fact that for orders that are beyond the `delivery` step, it's actually preferable to have the shipment's manifest be taxed for contents rather than the line items. The shipments know where they will be shipped from, making correct taxation for multiple stock locations possible.